Exercise06/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise06
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var n = 100;
+
+            var numbers = Enumerable.Range(2, n);

```

Comment:
`Enumerable.Range(2, 100);` ==> `2 3 4 .... 100 101` ==> wrong

Reply:
But x<n :v

Reply:
You're introducing a bug and then fix same time.  Sad

Reply:
Don't sad. i had remove it :v